src/vaadin-form-layout.html

```diff
@@ -172,7 +172,7 @@
         static get properties() {
           return {
             /**
-             * @typedef ResponsiveStep
+             * @typedef FormLayoutResponsiveStep
```

Comment:
I thought about removing this typedef block but I just renamed it to match the new TS interface. I think it's good to keep this too as it can still hint the IDE when working with Polymer 2 version.